DataScience/IntroToPython/IntroToPythonPandas/blog.md

```diff
@@ -0,0 +1,215 @@
+# Intro to Python Pandas
```

Comment:
Name your blog something more interesting. There are already 500000000 blogs out there with "Intro to Pandas". What makes your blog interesting? What separates it? It seems like you are introducing Spyder IDE. It might be a good idea to say "Intro to Spyder IDE with Pandas"

Reply:
using what data set?